.storybook/preview.ts

```diff
@@ -24,6 +15,14 @@ const preview: Preview = {
       },
     },
   },
+
```

Comment:
Would love if you could keep the js comment in place from the old globals key :)